### DISCUSSION TITLE
SHAP permutation explainer with SVM regressor

### DISCUSSION THREAD

**Discussion by shoubhikraj** [shap/shap#2658] [Q&A]
I am trying to explain a SVM regressor model. However, the KernelExplainer takes too long to be useful. My training set has 2500 entries and test set has 630.

Is permutation explainer usable with SVM regressor? By that I mean does it make sense to use permutation explainer on SVM regressor?
